python/modelUtils.py

```diff
@@ -47,10 +56,9 @@ def get_callbacks(model_filepath=None):
 
         logger_fp = model_filepath + "_history.csv"
         CSVLogger = keras.callbacks.CSVLogger(filename=logger_fp, append=False)
-
-        return [CheckPoint, CSVLogger, EarlyStopping]
+        return [cb for cb in [CheckPoint, CSVLogger, EarlyStopping, Reduce] if cb is not None] + lr_callbacks + custom_callbacks
```

Comment:
Hmm this looks unnecessarily convoluted (same for the other one below). How about just having a `callbacks = []`, and add to the list with if statement? The only cb that could be None is `Reduce`.

Reply:
And can `Reduce` and `PrintLearningRate` be configured elsewhere and included together with other ones in `lr_callbacks` for better encapsulation?